### DISCUSSION TITLE
Why U8G2 doesn't support OLED ic driver ST7735 or ST77xx ?

### DISCUSSION THREAD

**Discussion by denohk** [olikraus/u8g2#1477] [Q&A]
### Why U8G2 doesn't support OLED ic driver ST7735 or ST77xx ?

**Comment by olikraus** [comment]
U8g2 is optimized for one-bit per pixel displays (=monochrome displays). It simply doesn't support RGB displays like the ST7735.

If you are interessted in the fonts, then maybe look at https://github.com/olikraus/U8g2_for_Adafruit_GFX which works together with the Adafruit driver for the ST7735

Or maybe use my old lib (which is not supported any more but should still work):
https://github.com/olikraus/ucglib
